VideoLocker/src/main/java/org/edx/mobile/module/db/impl/IDatabaseImpl.java

```diff
@@ -32,7 +33,9 @@ public IDatabaseImpl(Context context, LoginPrefs loginPrefs) {
 
     @Nullable
     private String username() {
-        return loginPrefs.getUsername();
+        final String username = loginPrefs.getUsername();
+        if (username != null) return Sha1Util.SHA1(username);
```

Comment:
My preference here would actually be to just change the `username` variable itself:

```` java
if (username != null) {
    username = Sha1Util.SHA1(username);
}
return username;
````

This seems to be the clearest way to write it. But I'm fine with however you want to do it.